### ISSUE TITLE
preparation for more email output formats: separate one format per file

### ISSUE THREAD

**Issue by bernhardreiter** [Intevation/intelmq-mailgen#9]
In order to prepare intelmq-mailgen for more output formats,
e.g. for more schemas from xarf and other types as malware for csv,
we should put each format into a file (or module) of its own.

**Comment by dmth** [comment]
> and other types as malware for csv

Since https://github.com/Intevation/intelmq-mailgen/commit/2a5d08b3124dd7d6550c849260f8c754c71f7816 a default case exists for which enables this

**Comment by bernhardreiter** [comment]
@dmth to be extra clear: this issue is about the ability to have one (or several) formatters in one separate file. The default case of 2a5d08b is good, but does not help much towards that cause.

**Comment by dmth** [comment]
Yes. My intention was to remove the impression that only the type "malware" is supported right now, because two types exist at the moment: "Malware" and a default case.
